### PR TITLE
Apoc price armor and books

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

Summary: Content "Apoc Price armor and books"

#### Purpose of change

Listed in newcomer projects

#### Describe the solution

This is my second attempt as I'm trying to not recommit the broken syntax from the ammo files I used for my first PR.

#### Describe alternatives you've considered

This is my alternative I'm trying.

#### Additional context

I hope this works.

### CODE REVIEW

**Comment by Maleclypse** [comment]
I've gone through Lint with all the armor files and I'm working my way through the book files tonight.

File: data/json/items/armor/ammo_pouch.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "description": "A fabric ammo bag worn at the waist which is capable of holding a single large magazine close at hand.",
     "weight": "180 g",
     "volume": "500 ml",
-    "price": 5000,
+    "price": 2000,
+    "price_postapoc": 5000,
```
Comment (kevingranade) [inline data/json/items/armor/ammo_pouch.json]:
```suggestion
    "price_postapoc": 5000,
```

File: data/json/items/armor/ammo_pouch.json [hunk 2]
```diff
@@ -32,7 +33,8 @@
     "description": "A fabric ammo pouch that can be strapped to your chest which is capable of holding a single magazine close at hand.",
     "weight": "140 g",
     "volume": "250 ml",
-    "price": 2500,
+    "price": 1250,
+    "price_postapoc": 2500,
```
Comment (kevingranade) [inline data/json/items/armor/ammo_pouch.json]:
```suggestion
    "price_postapoc": 2500,
```

**Review by kevingranade — CHANGES_REQUESTED** [review]

File: data/json/items/armor/ammo_pouch.json (hunk 2 above)
Comment (Maleclypse) [reply data/json/items/armor/ammo_pouch.json]: I'm still learning so does the highlight mean you added a space or deleted a space?

File: data/json/items/book/cooking.json [hunk 3]
```diff
@@ -210,7 +221,9 @@
     "description": "This team logbook details several varieties of mutagenic experiments, focusing on those derived from various Earth fauna.  The team seems quite enthusiastic--if not eager--about their results.",
     "weight": "1700 g",
     "volume": "500 ml",
-    "price": 50000,
+    "price": 50000, 
+    "price_postapoc": 1250000,
+
```
Comment (kevingranade) [inline data/json/items/book/cooking.json]:
```suggestion
```

File: data/json/items/book/cooking.json [hunk 4]
```diff
@@ -210,7 +221,9 @@
     "description": "This team logbook details several varieties of mutagenic experiments, focusing on those derived from various Earth fauna.  The team seems quite enthusiastic--if not eager--about their results.",
     "weight": "1700 g",
     "volume": "500 ml",
-    "price": 50000,
+    "price": 50000,
```
Comment (kevingranade) [inline data/json/items/book/cooking.json]:
```suggestion
    "price": 50000,
```

**Review by kevingranade — CHANGES_REQUESTED** [review]

**Pull request merged**